lib/src/impl/realtime/channels.dart

```diff
@@ -48,10 +47,8 @@ class RealtimePlatformChannel extends PlatformObject
 
   Map<String, dynamic> __payload;
 
-  Map<String, dynamic> get _payload => __payload ??= {
-        'channel': name,
-        if (options != null) 'options': options
-      };
+  Map<String, dynamic> get _payload =>
+      __payload ??= {'channel': name, if (options != null) 'options': options};
```

Comment:
A trailing `,` after `options` would format this collection vertically like it was before.

Reply:
This commit fixes all trailing comma feedback: f5aa8d6438076b16eb6a06173f0b0b366acde2f8